### ISSUE TITLE
Clear wording

### ISSUE THREAD

**Issue by SigbjornR** [w3c/wcag3#232]
We would like to applaud the new focus on clear wording. By introducing this, WCAG will cover a wider selection of needs than today’s versions. 

It can however be challenging to (automatically) test clear wording and secure legal certainty. 

EU’s Web Accessibility Directive refers indirectly to WCAG and requires a high number of tests. To be able to comply with this, it must be possible to test automatically or semi-automatically. 

Moreover, many countries, including Norway, can enforce the requirements by fining the owner of the website, mobile application etc. As WCAG has become basis for legal requirements, and therefore also legal reactions in cases of breach, legal certainty becomes important when developing new requirements. 

We will therefore encourage the working group to dig deeper into how clear wording requirements can be evaluated in an objective, and preferably automatic, way. We are aware that there is an ISO-standard to come on clear wording (ISO/WS 24495-1), but we do not know whether is supports automatic testing.

The different needs between languages can make it challenging to develop a generic test. If there is no adequate readability measure, and it cannot be developed by W3C, we would like to call for the possibility for national adjustments through for example adding lists of words and phrases which should be avoided.

This will secure both legal certainty and more efficient enforcement.
